src/main/java/com/alipay/oceanbase/hbase/OHTable.java

```diff
@@ -365,44 +366,114 @@ public Object[] batch(List<? extends Row> actions) {
         throw new FeatureNotSupportedException("not supported yet.");
     }
 
-    public Result get(final Get get) throws IOException {
+    public byte[][] extractFamilyFromQualifier(byte[] qualifier) throws Exception {
+        int total_length = qualifier.length;
+        int familyLen = -1;
+        byte[][] res = new byte[2][];
+
+        for (int i = 0; i < total_length; i ++) {
+            if (qualifier[i] == '\0') {
+                familyLen = i;
+                break;
+            }
+        }
+
+        byte[] family = new byte[familyLen];
+        if (familyLen != -1) {
+            System.arraycopy(qualifier, 0 /* Skip delimiter */, family, 0, familyLen);
+        } else {
+            throw new RuntimeException("can not get family name");
+        }
+        res[0] = family;
+        int qualifierLen = total_length - familyLen - 1;
+        byte[] newQualifier = new byte[qualifierLen];
+        if (qualifierLen > 0) {
+            System.arraycopy(qualifier, familyLen + 1 /* Skip delimiter */, newQualifier, 0, qualifierLen);
+        } else {
+            throw new RuntimeException("can not get qualifier name");
+        }
+        res[1] = newQualifier;
+        return res;
+    }
 
-        checkFamilyViolation(get.getFamilyMap().keySet());
+    public void getKeyValueFromResult(ObTableClientQueryStreamResult clientQueryStreamResult, 
```

Comment:
What about ObTableClientQueryAsyncStreamResult

Reply:
A single ObTableClientQueryAsyncStreamResult handles the same, use next() and getRow().